src/index.html

```diff
@@ -9,10 +9,43 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/style.scss"
```

Comment:
The stylesheet link references a `.scss` file, which is a Sass file. Ensure this file is compiled to a `.css` file and link the resulting `.css` file instead.